plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/_config.py

```diff
@@ -39,6 +39,86 @@ class RayAutoScalingMode(Enum):
     aggressive = "aggressive"
 
 
+class RayLoggingStyle(Enum):
+    """
+    If 'pretty', outputs with formatting and color.
+    If 'record', outputs record-style without formatting.
+    'auto' defaults to 'pretty', and disables pretty logging
+    if stdin is *not* a TTY. Defaults to "auto".
+    """
+
+    auto = "auto"
+    pretty = "pretty"
+    record = "record"
+
+
+class RayLoggingColorMode(Enum):
+    """
+    Enables or disables `colorful`. Can be "true", "false", or "auto".
+    """
+
+    true = "true"
+    false = "false"
+    auto = "auto"
+
+
+class RayLoggingVerbosity(IntEnum):
+    """
+    Low verbosity will disable `verbose` and `very_verbose` messages.
+    """
+
+    minimal = 0
+    verbose = 1
+    very_verbose = 2
+    very_very_verbose = 3
+
+
+@dataclass
+class RayLoggingConf:
+    """
+    Ray sdk.configure_logging parameters: log_style, color_mode, verbosity
+    """
+
+    log_style: RayLoggingStyle = RayLoggingStyle.auto
+    color_mode: RayLoggingColorMode = RayLoggingColorMode.auto
+    verbosity: int = RayLoggingVerbosity.verbose.value
```

Comment:
FYI, I originally wanted this to be `verbosity: RayLoggingVerbosity = RayLoggingVerbosity.verbose`. `OmegaConf.to_container` with `enum_to_str` gives `verbosity: "verbose"` instead of `verbosity: 1` and hence had to adopt the current solution. If you have a better solution, I'm happy to change this.

Reply:
It looks like there isn't much difference between [minimal](https://gist.github.com/syed-ahmed/27c64c0ff0fdc2eec6d4a45406a3119a) and [verbose](https://gist.github.com/syed-ahmed/c3b4fda9054155092b922bf2e8d32d34), and ray still outputs a lot at `minimal` logging. I additionally print out the cluster config. So may be I'll turn the cluster config logging to debug and the verbosity to minimal?

Reply:
Yea, i think we can skip printing out the cluster config (users can get the config from the app via Hydra cli `--cfg hydra --resolve`)
Let's keep the default logging minimal then :)